sharkclipper/api/handlers.py

```diff
@@ -5,6 +5,7 @@
 import logging
 import mimetypes
 import os
+import shutil
```

Comment:
Instead of `shutil.rmtree()`, you should use `sharkclipper.util.file.remove_dirent()`.
In this case they will work the same, but it is nice to be consistent with the tools we use.